examples/main/views/custom-interaction.js

```diff
@@ -62,7 +64,7 @@ const propTypes = {
   height: PropTypes.number.isRequired
 };
 
-export default class TiltExample extends Component {
+export default class Example extends Component {
```

Comment:
Nit: I believe that react automatically sets display name based on class name, at least in unminified code this can be helpful during debugging...